core/core-impl/src/main/java/io/fabric8/launcher/core/impl/filters/SecuredFilter.java

```diff
@@ -55,16 +59,16 @@ public void filter(ContainerRequestContext requestContext) {
         }
     }
 
-    private void abortWithUnauthorized(ContainerRequestContext requestContext) {
-
-        // Abort the filter chain with a 401 status code response
-        requestContext.abortWith(
-                Response.status(Response.Status.UNAUTHORIZED)
-                        .build());
+    private DecodedJWT validateToken(String token) {
+        final DecodedJWT jwt = JWT.decode(token);
+        final JWTValidator jwtValidator = new JWTValidator(jwt.getIssuer(), publicKeyProvider);
+        if (!jwtValidator.validate(token)) {
+            throw new IllegalArgumentException("Invalid token");
```

Comment:
@gastaldi Shall we pass it as error to Sentry? :fire: :fire: :bomb: :fireworks:

Reply:
Doesn't look like it needs to. If the token is invalid, there isn't much we can do about it